opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -93,10 +93,14 @@ option go_package = "go.opentelemetry.io/proto/otlp/profiles/v1development";
 // ProfilesDictionary represents the profiles data shared across the
 // entire message being sent.
 message ProfilesDictionary {
+
+  // Note all fields in this message MUST have a zero value encoded as the first element.
```

Comment:
Should we also add this to make completely unambiguous?

```suggestion
  // All fields in this message MUST be present and MUST have a zero value encoded as the first element.
```

Reply:
I don't think that's needed. Requiring the first element in a repeated list to be present implies the field to be present. Resolving this to make the PR mergable. Feel free to raise a new PR if you want to push this forward.